autolight.ventura.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Begin looking at the system log via the steam sub-command. Using a --predicate and filtering by the correct and pull out the camera event 
+log stream --predicate 'subsystem == "com.apple.cmio" and composedMessage contains "streaming for client"' | while read line; do
```

Comment:
I don't know if it's more or less efficient, but I also found success with `"composedMessage CONTAINS "Power" AND composedMessage CONTAINS "Camera" AND sender == "appleh13camerad""` FWIW.

Reply:
Thanks for this comment, I would imagine the differences are negligible. Have you tested on Sonoma?